sqlglot/diff.py

```diff
@@ -0,0 +1,162 @@
+from dataclasses import dataclass
+from sqlglot.expressions import (
+    Anonymous,
+    Column,
+    Connector,
+    DataType,
+    Expression,
+    Group,
+    Identifier,
+    In,
+    Literal,
+    Select,
+    Table,
+    TableAlias,
+    With,
+)
+from sqlglot.errors import DiffError
+from sqlglot.helper import ensure_list
+
+
+@dataclass
+class Insert:
+    expr: Expression
```

Comment:
minor, but can we just call it expression: Expression 

and for keep source / target, you have the types there anyways, so no need to say expr_src

Reply:
yes, just to source and target, it's clear it's an expression

Reply:
Fixed this